Services/CO.CDP.Localization/StaticTextResource.resx

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!--
-    Microsoft ResX Schema
-
+  <!-- 
+    Microsoft ResX Schema 
+    
```

Comment:
All these whitespaces at the end of lines should be reverted.

Reply:
@JBaigGoaco Did you edit this solely with the visual studio resource manager? It's quite likely this whitespace is the "correct" whitespace - this diff is probably caused by the translation script, I really struggled to make it match.

Basically I wonder whether to let the whitespace changes slip in this time @jakzal - and I try again to address the issues caused by the script next time we use it.